src/chttpd/src/chttpd.erl

```diff
@@ -1111,14 +1111,16 @@ error_headers(#httpd{mochi_req=MochiReq}=Req, 401=Code, ErrorStr, ReasonStr) ->
             % redirect to the session page.
             case ErrorStr of
             <<"unauthorized">> ->
-                case config:get("couch_httpd_auth", "authentication_redirect", undefined) of
+                case chttpd_util:get_chttpd_auth_config(
+                    "authentication_redirect", "/_utils/session.html") of
```

Comment:
please use `undefined` as default here to avoid breaking users installations during upgrade.

Reply:
Actually never mind. We can update the default on `main` because it was not released yet. However for 3.x we should preserve existent behavior.